YD-22 widgets premium (#152)
YetiForceCompany/YetiForceDoc: docs/3-administrator-guides/04-marketplace/02-addons/02-widgets-premium/README.md

---
title: Widżety Premium
---

**widżety Premium**  pozwalają na wyświetlenie na pulpicie i w podsumowaniu rekordu dodatkowych informacji, dzięki czemu wszystkie niezbędne szczegóły mogą być dostępne dla użytkownika na wyciągnięcie ręki. W chwili obecnej w zestawie dostępne są 2 widżety - **Podgląd PDF** oraz **Historia zmian**

:::warning

> widżet Podgląd PDF i Historia Zmian są dostępne tylko dla użytkowników YetiForce CRM z wykupioną subskrypcją widżety Premium.

Reviewer: ```suggestion
Widżety Premium są dodatkiem płatnym i wymagają wykupienia subskrypcji.
```

:::

## Podgląd PDF

> Jest to widżet, który pozwala na podgląd dokumentu PDF do wygenerowania w czasie rzeczywistym. Dzięki temu będziesz miał możliwość podglądu faktury, lub innego dokumentu PDF, przed jego fizycznym wygenerowaniem. Takie rozwiązanie znacznie ułatwi pracę użytkownikom odpowiedzialnym za wystawianie i przesyłanie dokumentów PDF, oraz pracownikom działu księgowego, gdyż umożliwia weryfikacje dokumentu przed jego wystawieniem.

Reviewer: ```suggestion
Jest to widżet, który pozwala na podgląd dokumentu PDF do wygenerowania w czasie rzeczywistym. Dzięki temu będziesz miał możliwość podglądu faktury lub innego dokumentu PDF przed jego fizycznym wygenerowaniem. Takie rozwiązanie znacznie ułatwi pracę użytkownikom odpowiedzialnym za wystawianie i przesyłanie dokumentów PDF oraz pracownikom działu księgowego, gdyż umożliwia weryfikacje dokumentu przed jego wystawieniem.
```

### Konfiguracja Podglądu PDF

W pierwszej kolejności należy się upewnić, czy dla danego modułu posiadamy stworzony szablon PDF.
Następnie możemy przejść do konfiguracji samego widżetu:

> 1. Przejdź do panelu administracyjnego YetiForce CRM.

Reviewer: ```suggestion
1. Przejdź do panelu administracyjnego YetiForce.
```

2. Wybierz z lewego menu  Moduły standardowe ➜ Moduły widżety.
3. Z pola wyboru, po prawej stronie wybierz moduł, dla którego chcesz dodać widżet **Podgląd PDF**.

> 4. Kliknij przycisk Dodaj widżet.

Reviewer: ```suggestion
4. Kliknij przycisk **Dodaj widżet**.
```

5. Wybierz widżet **Podgląd PDF** z listy dostępnych widżetów
6. Wprowadź nazwę widżetu i zapisz zmiany

> ![widgets-1.jpg](widgets-1.jpg)

Reviewer: ten zrzut nie jest spójny z następnym. W tym tym obrazujesz dodanie go do Leada, a w następnym masz zrzut z modułu faktur.
proszę zachowuj ciągłość procesu, aby nie mylić czytelników ;)

Widżet **Podgląd PDF** jest już gotowy do użycia. W panelu użytkownika, przejdź do modułu dla którego go dodałeś a następnie wybierz jeden z rekordów. Widżet powinien pojawić się na podsumowaniu rekordu.

![widgets-2.jpg](widgets-2.jpg)

## Historia zmian

> Jest to widżet, który umożliwia wgląd w historię aktywności na rekordach modułów. Pozwala on na weryfikacje konkretnej aktywności dowolnych użytkowników w obrębie wybranego modułu bądź globalnie. Wyświetla on listę rekordów, które były otwierane lub modyfikowane przez użytkowników, czas przeprowadzanej aktywności, oraz użytkownika, który danej operacji dokonał. widżet zawiera liczne zaawansowane filtry, które pozwolą zawęzić wyświetlaną treść do interesujących nas wyników.

Reviewer: ```suggestion
Jest to widżet, który umożliwia wgląd w historię aktywności na rekordach modułów. Pozwala on na weryfikacje konkretnej aktywności dowolnych użytkowników w obrębie wybranego modułu bądź globalnie. Wyświetla on listę rekordów, które były przeglądane lub modyfikowane w zadanym przedziale czasu. Widżet zawiera liczne zaawansowane filtry, które pozwolą zawęzić wyświetlaną treść do interesujących nas wyników.
```

### Konfiguracja Historii Zmian

:::warning

Pamiętaj by przy tworzeniu widżetu wybrać odpowiednią rolę oraz pulpit, na którym ma się on pojawić.
:::

> 1. Przejdź do panelu administracyjnego YetiForce CRM.

Reviewer: ```suggestion
1. Przejdź do panelu administracyjnego YetiForce.
```

2. Wybierz z lewego menu  Moduły standardowe ➜ Panel widżety.

> 4. Kliknij przycisk Predefiniowane widżety.

Reviewer: ```suggestion
4. Kliknij przycisk **Predefiniowane widżety**.
```

> 5. Wybierz widżet **Historia aktualizacji systemu** z listy dostępnych widżetów

Reviewer: ```suggestion
5. Wybierz widżet **Historia aktualizacji systemu** z listy dostępnych widżetów.
```

> 6. Wprowadź parametry widżetu i zapisz zmiany

Reviewer: ```suggestion
6. Wprowadź parametry widżetu i zapisz zmiany.
```

![widgets-3.jpg](widgets-3.jpg)

Widżet **Historia aktualizacji systemu** jest już gotowy do użycia. W panelu użytkownika przejdź do Zakładki **Pulpit**,  gdzie pojawi się widżet. 

Pierwsza lista rozwijana pozwala nam wybrać moduł systemu, dla którego chcemy śledzić zmiany. Jeżeli z listy wybierzemy pozycję **Podsumowanie**, pojawi nam się historia aktualizacji w wybranym okresie dla całego systemu z podziałem na moduły.

> ![widgets-4.jpg](widgets-4.jpg)

Reviewer: nadal słabo zobrazowany widget. w konfiguracji masz do dyspozycji kilkanaście opcji aktywności, a prezentujesz tylko 3

> Wybierając natomiast z listy konkretny moduł, uzyskamy dodatkowe informacje, takie jak: kto wykonał akcję, na jakim rekordzie czy też z jakich danych na jakie zostały zmienione

Reviewer: ```suggestion
Wybierając z listy konkretny moduł uzyskujemy od razu bardziej szczegółowe, jak np. kto, kiedy i jaką aktywność przeprowadził.
```

![widgets-5.jpg](widgets-5.jpg)

> W prawym górnym rogu widżetu znajduje się ikona trybika, pozwalająca ustawić wyświetlane przez niego dane. Możemy wybrać jakie akcje chcemy, żeby zostały wyśwetlone oraz z jakimi użytkownikami mają być one powiązane.

Reviewer: ```suggestion
W prawym górnym rogu widżetu znajduje się ikona trybika, pozwalająca ustawić zakres wyświetlanych danych. Mamy do dyspozycja całą masę aktywności z dodatkową możliwością zawężania do konkretnych użytkowników.
```
